win32ss/printing/base/winspool/printers.c

```diff
@@ -386,8 +386,38 @@ EndPagePrinter(HANDLE hPrinter)
 BOOL WINAPI
 EnumPrintersA(DWORD Flags, PSTR Name, DWORD Level, PBYTE pPrinterEnum, DWORD cbBuf, PDWORD pcbNeeded, PDWORD pcReturned)
 {
+    WCHAR NameW[255];
+    CHAR  NameA[255];
+    BOOL ret;
+    INT i;
+    PPRINTER_INFO_4W ppi4w = NULL;
+    PPRINTER_INFO_4A ppi4a = NULL;
+
     TRACE("EnumPrintersA(%lu, %s, %lu, %p, %lu, %p, %p)\n", Flags, Name, Level, pPrinterEnum, cbBuf, pcbNeeded, pcReturned);
-    return FALSE;
+    if (Name == NULL)
+        NameW[0] = UNICODE_NULL;
+    else
+    /* https://stackoverflow.com/questions/41147180/why-enumprintersa-and-enumprintersw-request-the-same-amount-of-memory */
+        MultiByteToWideChar(CP_ACP, 0, Name, -1, NameW, 255);
```

Comment:
Add `{ }` against `if` block and `else` block.

Reply:
Done.